influxql/ast.go

```diff
@@ -2822,6 +2822,10 @@ func Walk(v Visitor, node Node) {
 			Walk(v, c)
 		}
 
+	case *DropSeriesStatement:
```

Comment:
Some statement types don't need to be represented as a `case` because they have no members that can be walked.  If we added a `panic` then each of those would have to have an empty `case`.  But, doing that still wouldn't prevent this type of bug because we still have to remember to update the `Walk` function if a walkable member is added to one of the types.